libsql-server/src/stats.rs

```diff
@@ -52,6 +53,100 @@ impl SlowestQuery {
     }
 }
 
+#[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub struct QueryStats {
+    pub elapsed_ms: u64,
+    pub count: u64,
+    pub rows_written: u64,
+    pub rows_read: u64,
+}
+
+impl QueryStats {
+    pub fn new(elapsed_ms: u64, rows_read: u64, rows_written: u64) -> Self {
+        Self {
+            elapsed_ms,
+            count: 1,
+            rows_read,
+            rows_written,
+        }
+    }
+    pub fn empty() -> Self {
+        Self {
+            elapsed_ms: 0,
+            count: 0,
+            rows_read: 0,
+            rows_written: 0,
+        }
+    }
```

Comment:
I believe default will do this just fine? no need for a second fn?

Reply:
we don't,  this is a leftover from when I stored the SQL query itself inside the stats struct
back then we needed a default + SQL
I'll remove it!